.pre-commit-config.yaml

```
@@ -1,12 +1,16 @@
+ci:
+  autofix_prs: false
+  autoupdate_schedule: monthly
```

```suggestion
  autoupdate_schedule: quarterly
```
...in case you prefer less frequent auto-updates.